### DISCUSSION TITLE
star exchange

### DISCUSSION THREAD

**Discussion by larry-xue** [drknzz/GitHub-Achievements#352] [General]
star my repo: https://github.com/larry-xue/larry-xue-site

and I will star your repo back.

**Comment by Denisyasyuchenya** [comment]
https://github.com/Denisyasyuchenya/my-location-app this one please

**Reply by larry-xue** [reply]
done

**Reply by jvcByte** [reply]
@Denisyasyuchenya done. star this https://github.com/jvc-byte/Simple-Bank-Using-solidity please

**Reply by rumiantsevaa** [reply]
Done;)for both of you this one please https://github.com/rumiantsevaa/Face_Recognition_Nancy_Database

**Reply by dorienh** [reply]
Did yours too. Here is mine: https://github.com/AMAAI-Lab/Text2midi

**Reply by VArtzy** [reply]
done https://github.com/VArtzy/neocurl

**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours

**Reply by VArtzy** [reply]
done @AyushKatre05  https://github.com/VArtzy/neocurl

**Reply by AyushKatre05** [reply]
@VArtzy  done

**Reply by MatteoMgr2008** [reply]
@larry-xue
@Denisyasyuchenya 
star mine i stared yours:
https://github.com/MatteoMgr2008/Arduino-ROBOTICA-E-AI

**Reply by harsh-kakadiya1** [reply]
star me and i will start yours 
https://github.com/harsh-kakadiya1/datamimic.io

**Reply by Istituto-freudinttheprodev** [reply]
@harsh-kakadiya1
star mine please I stared yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Reply by harsh-kakadiya1** [reply]
done @Istituto-freudinttheprodev

**Reply by Istituto-freudinttheprodev** [reply]
> done @Istituto-freudinttheprodev 
> 

You're the best! Thank you so much @harsh-kakadiya1

**Reply by malmorox** [reply]
done!
star mine i stared yours:
https://github.com/malmorox/Portfolio

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by MrArnaudMichel** [comment]
https://github.com/MrArnaudMichel/Coco-Bot this one please !

**Reply by larry-xue** [reply]
done

**Reply by jvcByte** [reply]
@MrArnaudMichel done, this https://github.com/jvc-byte/Simple-Bank-Using-solidity please

**Reply by MrArnaudMichel** [reply]
Done

**Reply by rumiantsevaa** [reply]
Done, this one please https://github.com/rumiantsevaa/Face_Recognition_Nancy_Database

**Reply by dorienh** [reply]
Done https://github.com/AMAAI-Lab/Text2midi

**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours

**Reply by MatteoMgr2008** [reply]
@MrArnaudMichel
star mine i stared yours:
https://github.com/MatteoMgr2008/Arduino-ROBOTICA-E-AI

**Reply by malmorox** [reply]
done!
star mine i stared yours:
https://github.com/malmorox/Portfolio

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by KamBRJC** [comment]
https://github.com/z-nnovation/GitSwitcher this one please

**Reply by larry-xue** [reply]
done

**Reply by jvcByte** [reply]
@KamBRJC done. This https://github.com/jvc-byte/Simple-Bank-Using-solidity please

**Reply by rumiantsevaa** [reply]
Done;) this one please https://github.com/rumiantsevaa/Face_Recognition_Nancy_Database

**Reply by dorienh** [reply]
Done https://github.com/AMAAI-Lab/Text2midi

**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours

**Reply by MatteoMgr2008** [reply]
@KamBRJC
star mine i stared yours:
https://github.com/MatteoMgr2008/Arduino-ROBOTICA-E-AI

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by jvcByte** [comment]
@larry-xue done. This please https://github.com/jvc-byte/Simple-Bank-Using-solidity

**Reply by larry-xue** [reply]
done

**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by HuluWZ** [comment]
star my repo:  https://github.com/HuluWZ/phone-formatter-eth

and I will star your repo back.

It's npm package for mobile phone formatting package. https://www.npmjs.com/package/phone-formater-eth

**Reply by rumiantsevaa** [reply]
Done! Please star this one https://github.com/rumiantsevaa/Face_Recognition_Nancy_Database

**Reply by larry-xue** [reply]
done

**Reply by HuluWZ** [reply]
> done

make sure the repo is
https://github.com/HuluWZ/phone-formatter-eth @larry-xue

**Reply by dorienh** [reply]
Done https://github.com/AMAAI-Lab/Text2midi

**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours

**Reply by SandunMunasinghe20** [reply]
@HuluWZ  done https://github.com/SandunMunasinghe20/job-portal-springboot-react

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by rumiantsevaa** [comment]
Star these repos please: 
Any amount is mutual⭐️Like one or several repos, if you have the same amount of repos 1=1, 4=4, 11=11
https://github.com/rumiantsevaa/Face_Recognition_Nancy_Database


https://github.com/rumiantsevaa/Telescopy_Telegram_Bot

https://github.com/rumiantsevaa/DIXIT_Telegram_Bot

https://github.com/rumiantsevaa/Bloodhound_Telegram_Bot

**Reply by HuluWZ** [reply]
done

**Reply by rumiantsevaa** [reply]
Starred yours already:)

**Reply by dorienh** [reply]
Done https://github.com/AMAAI-Lab/Text2midi

**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours

**Reply by SandunMunasinghe20** [reply]
https://github.com/SandunMunasinghe20/job-portal-springboot-react

**Reply by khalifarsm** [reply]
https://github.com/khalifarsm/botox-android

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by ELHart05** [comment]


**Reply by rumiantsevaa** [reply]
DONE, THX

**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours

**Reply by khalifarsm** [reply]
https://github.com/khalifarsm/botox-android

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by VArtzy** [comment]


**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours

**Reply by rumiantsevaa** [reply]
@AyushKatre05  starred yours star mine pls https://github.com/rumiantsevaa/lunar_majesty

**Reply by khalifarsm** [reply]
https://github.com/khalifarsm/botox-android

**Reply by SandunMunasinghe20** [reply]
@AyushKatre05  done     https://github.com/SandunMunasinghe20/job-portal-springboot-react

**Reply by SandunMunasinghe20** [reply]
@VArtzy  done.    https://github.com/SandunMunasinghe20/job-portal-springboot-react

**Reply by malmorox** [reply]
done https://github.com/malmorox/Portfolio star mine pls

**Reply by AyushKatre05** [reply]
@malmorox  follow me u will star anyhow

**Reply by shaaravraghu** [reply]
> https://github.com/khalifarsm/botox-android

Please star this: https://github.com/shaaravraghu/ClipWallet/

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by rumiantsevaa** [comment]
Mutual stars, guys! ⭐️⭐️⭐️
Star mine and send me yours to star!
Chose how much ⭐️⭐️REPO STARS ⭐️⭐️you want to exchange, star ⭐️ mine and send me fair amount of your links to star in exchange

1=1 ⭐️
https://github.com/rumiantsevaa/lunar_majesty

2=2 ⭐️⭐️
https://github.com/rumiantsevaa/lunar_majesty
https://github.com/rumiantsevaa/PurgeAllButCats

3=3 ⭐️⭐️⭐️
https://github.com/rumiantsevaa/Diploma-Project-Booking-Service
https://github.com/rumiantsevaa/lunar_majesty
https://github.com/rumiantsevaa/PurgeAllButCats

4=4 ⭐️⭐️⭐️⭐️
https://github.com/rumiantsevaa/Diploma-Project-Booking-Service
https://github.com/rumiantsevaa/lunar_majesty
https://github.com/rumiantsevaa/PurgeAllButCats
https://github.com/rumiantsevaa/Telescopy_Telegram_Bot

**Reply by SandunMunasinghe20** [reply]
https://github.com/SandunMunasinghe20/job-pulse

**Reply by khalifarsm** [reply]
> Mutual stars, guys! ⭐️⭐️⭐️ Star mine and send me yours to star! Chose how much ⭐️⭐️REPO STARS ⭐️⭐️you want to exchange, star ⭐️ mine and send me fair amount of your links to star in exchange
> 
> 1=1 ⭐️ https://github.com/rumiantsevaa/lunar_majesty
> 
> 2=2 ⭐️⭐️ https://github.com/rumiantsevaa/lunar_majesty https://github.com/rumiantsevaa/PurgeAllButCats
> 
> 3=3 ⭐️⭐️⭐️ https://github.com/rumiantsevaa/Diploma-Project-Booking-Service https://github.com/rumiantsevaa/lunar_majesty https://github.com/rumiantsevaa/PurgeAllButCats
> 
> 4=4 ⭐️⭐️⭐️⭐️ https://github.com/rumiantsevaa/Diploma-Project-Booking-Service https://github.com/rumiantsevaa/lunar_majesty https://github.com/rumiantsevaa/PurgeAllButCats https://github.com/rumiantsevaa/Telescopy_Telegram_Bot

https://github.com/khalifarsm/botox-android

**Reply by rumiantsevaa** [reply]
Did star both of you, https://github.com/SandunMunasinghe20/job-pulse AND https://github.com/khalifarsm/botox-android . 

@SandunMunasinghe20 PLS STAR mine too https://github.com/rumiantsevaa/Telescopy_Telegram_Bot

**Reply by hasseneafif** [reply]
Hey guys i have starred most of your repos, star 4 of my projects, also will star anyone new that stars me
https://github.com/hasseneafif/ai-performance-check
https://github.com/hasseneafif/expensy
https://github.com/hasseneafif/ailex
https://github.com/hasseneafif/ai-news-framer

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by harsh-kakadiya1** [comment]
star my repo and i will star yours
https://github.com/harsh-kakadiya1/datamimic.io
comment 
done @harsh-kakadiya1

**Reply by Istituto-freudinttheprodev** [reply]
done, I starred your repo @harsh-kakadiya1 !

**Reply by rumiantsevaa** [reply]
done @harsh-kakadiya1 now star mine pls https://github.com/rumiantsevaa/Telescopy_Telegram_Bot

**Reply by harsh-kakadiya1** [reply]
Done @rumiantsevaa

**Reply by Istituto-freudinttheprodev** [reply]
@harsh-kakadiya1
star mine i stared yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Reply by harsh-kakadiya1** [reply]
done @Istituto-freudinttheprodev

**Reply by Istituto-freudinttheprodev** [reply]
Thank you so much @harsh-kakadiya1

**Reply by SandunMunasinghe20** [reply]
https://github.com/SandunMunasinghe20/job-portal-springboot-react

star this then i will do the same

**Reply by khalifarsm** [reply]
> @harsh-kakadiya1 star mine i stared yours: https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

https://github.com/khalifarsm/botox-android

**Reply by harsh-kakadiya1** [reply]
@khalifarsm done bro

**Reply by rumiantsevaa** [reply]
> https://github.com/SandunMunasinghe20/job-pulse
> 
> star this then i will do the same
@SandunMunasinghe20  done! now star mine pls https://github.com/rumiantsevaa/Telescopy_Telegram_Bot

**Reply by khalifarsm** [reply]
done @rumiantsevaa @SandunMunasinghe20

**Reply by SandunMunasinghe20** [reply]
@harsh-kakadiya1  done.

**Reply by harsh-kakadiya1** [reply]
@SandunMunasinghe20 bro done i starred yours!

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by khalifarsm** [comment]
https://github.com/khalifarsm/botox-android

**Reply by harsh-kakadiya1** [reply]
Done bro

**Reply by SandunMunasinghe20** [reply]
done

**Reply by rumiantsevaa** [reply]
done @khalifarsm now star mine pls https://github.com/rumiantsevaa/Telescopy_Telegram_Bot

**Reply by khalifarsm** [reply]
done

**Reply by malmorox** [reply]
done! https://github.com/malmorox/Portfolio star mine pls

**Reply by wongkafai-teacher** [reply]
Done! Starred your repository. Here is my educational profile repository, please star back: [https://github.com/wongkafai-teacher/wongkafai]

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by rumiantsevaa** [comment]
Mutual stars, guys! ⭐️⭐️⭐️
Star mine and send me yours to star!
Chose how much ⭐️⭐️REPO STARS ⭐️⭐️you want to exchange, star ⭐️ mine and send me fair amount of your links to star in exchange

1=1 ⭐️
https://github.com/rumiantsevaa/lunar_majesty

2=2 ⭐️⭐️
https://github.com/rumiantsevaa/lunar_majesty
https://github.com/rumiantsevaa/PurgeAllButCats

3=3 ⭐️⭐️⭐️
https://github.com/rumiantsevaa/Diploma-Project-Booking-Service
https://github.com/rumiantsevaa/lunar_majesty
https://github.com/rumiantsevaa/PurgeAllButCats

4=4 ⭐️⭐️⭐️⭐️
https://github.com/rumiantsevaa/Diploma-Project-Booking-Service
https://github.com/rumiantsevaa/lunar_majesty
https://github.com/rumiantsevaa/PurgeAllButCats
https://github.com/rumiantsevaa/Telescopy_Telegram_Bot

**Reply by malmorox** [reply]
done! https://github.com/malmorox/Portfolio

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by malmorox** [comment]
done https://github.com/malmorox/Portfolio

**Reply by Rooster157** [reply]
star mine i will star yours :https://github.com/ArshiBansal/Stocks_Analysis

**Reply by wongkafai-teacher** [reply]
Done! Starred your repository. Here is my educational profile repository, please star back: [https://github.com/wongkafai-teacher/wongkafai]

**Reply by Olayiwola72** [reply]
https://github.com/Olayiwola72/tablesmit

Left a star, please star back @Rooster157

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by hasseneafif** [comment]
Hey guys i have starred most of your repos, star 4 of my projects, also will star anyone new that stars me
https://github.com/hasseneafif/ai-performance-check
https://github.com/hasseneafif/expensy
https://github.com/hasseneafif/ailex
https://github.com/hasseneafif/ai-news-framer

**Reply by AyushKatre05** [reply]
follow me

**Reply by wongkafai-teacher** [reply]
Done! Starred your repository. Here is my educational profile repository, please star back: [https://github.com/wongkafai-teacher/wongkafai]

**Reply by Olayiwola72** [reply]
https://github.com/Olayiwola72/tablesmit

Left a star, please star back @hasseneafif

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by wongkafai-teacher** [comment]
Hey guys, neat profiles here! Check out my educational portfolio readme at (https://github.com/wongkafai-teacher/wongkafai). Leave a star on mine, reply here, and I'll star you back within an hour! Let's connect!"

**Reply by Olayiwola72** [reply]
https://github.com/Olayiwola72/tablesmit

Left a star, please check out mine

**Reply by Olayiwola72** [reply]
https://github.com/Olayiwola72/tablesmit

Left a star, please star back

**Comment by L1nkinPark** [comment]
https://github.com/h1eudayne/devops-ci-cd-templates
This one pls

**Reply by wongkafai-teacher** [reply]
If you star me, I will star you back. [https://github.com/wongkafai-teacher/wongkafai]

**Reply by Olayiwola72** [reply]
> https://github.com/h1eudayne/devops-ci-cd-templates This one pls

https://github.com/Olayiwola72/tablesmit

Left a star, please check out mine @L1nkinPark

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by Olayiwola72** [comment]
https://github.com/Olayiwola72/tablesmit

Star mine and I will star back

**Reply by eyalgolan** [reply]
@Olayiwola72 done, please star mine https://github.com/no-human-ai/no_human.
Thanks!

**Reply by Olayiwola72** [reply]
done @eyalgolan

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by YunosukeYoshino** [comment]
Drop your repo and I'll check it out too!
https://github.com/YunosukeYoshino/skillloom

**Reply by shaaravraghu** [reply]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Reply by eyalgolan** [reply]
@YunosukeYoshino I starred your repo, please check out and star no_human https://github.com/no-human-ai/no_human.

**Comment by shaaravraghu** [comment]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Comment by shaaravraghu** [comment]
Please star this: https://github.com/shaaravraghu/ClipWallet/

**Reply by eyalgolan** [reply]
@shaaravraghu I starred your repo, please check out and star no_human https://github.com/no-human-ai/no_human.

**Reply by zermello** [reply]
> @shaaravraghu I starred your repo, please check out and star no_human https://github.com/no-human-ai/no_human.

https://github.com/zermello/turing-ai-agent i starred yours please star my back!

**Reply by zermello** [reply]
> Please star this: https://github.com/shaaravraghu/ClipWallet/

Started yours can u star mine 
https://github.com/zermello/turing-ai-agent

**Comment by zermello** [comment]
Please star this:
https://github.com/zermello/turing-ai-agent
